pkg/operator/ceph/csi/csi.go

```diff
@@ -366,3 +367,26 @@ func (r *ReconcileCSI) setParams() error {
 
 	return nil
 }
+
+func (r *ReconcileCSI) deleteRookCSICMIfExists() error {
+	logger.Debug("checking for existing rook-ceph-csi-config configmap to delete...")
+
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ConfigName,
+			Namespace: r.opConfig.OperatorNamespace,
+		},
+	}
+
+	err := r.client.Delete(r.opManagerContext, cm)
+	if err != nil {
+		if kerrors.IsNotFound(err) {
+			logger.Debugf("configmap %q not found. Ignoring since it must already be deleted", ConfigName)
+			return nil
+		}
+		return errors.Wrapf(err, "failed to delete configmap %q in namespace %q", ConfigName, r.opConfig.OperatorNamespace)
+	}
+
+	logger.Debugf("successfully deleted configmap %q in namespace %q", ConfigName, r.opConfig.OperatorNamespace)
```

Comment:
```suggestion
	logger.Infof("successfully deleted configmap %q in namespace %q", ConfigName, r.opConfig.OperatorNamespace)
```

Reply:
Agreed to change this to Info level, but I would vote for the other logs to stay as debug since we don't need all the logging for the common case when the configmap does not exist.

Reply:
done